### ISSUE TITLE
Replace calls to ActiveFedora::SolrService.query (removes 9 refs to ActiveFedora)

### ISSUE THREAD

**Issue by elrayle** [samvera/hyrax#3801]
### Prerequisites

This issue cannot be completed until the following issues are completed first...

* #3800  setup solr indexers and metadata adapter

----
### Descriptive summary

Th `ActiveFedora::SolrService.query` is used to submit a query to solr.  The replacement for this is to use Valkyrie to connect to and submit queries to solr.

This issue involves removing references to `ActiveFedora::SolrService.query`.

### ActiveFedora code

Active Fedora call being replaced...

Example from app/search_builders/my/search_builder.rb
```
        ids = ActiveFedora::SolrService.query("{!field f=member_ids_ssim}#{id}",
                                              fl: ActiveFedora.id_field)
```

To locate, search code for `ActiveFedora::SolrService.query` in `app`.

### Replacement code

Example from app/search_builders/my/search_builder.rb
```
      solr = Valkyrie::MetadataAdapter.find(:index_solr).connection
      results = solr.get('select', params: { q: "{!field f=member_ids_ssim}id-#{id}",
                                             fl: 'id',
                                             qt: 'standard' })
```

For some instances, you can compare occurrences in [master app](https://github.com/samvera/hyrax/tree/master/app) with the corresponding occurrence in [valkryie-6 app](https://github.com/samvera/hyrax/tree/valkyrie-6/app) to see how it was changed in that branch.

### Considerations

* Not all diffs between master and valkryie-6 branch are necessarily related to removing references to ActiveFedora.  The valkyrie-6 branch has not been kept in sync with master.
* Do NOT replace this call if found in `lib/wings` directories.

### Impact on Testing

Not expected to require test changes.  All tests should pass with the replacement code.

**Comment by hackmastera** [comment]
The first phase of this is done in #3804.

**Comment by elrayle** [comment]
This has been addressed by Hyrax::SolrService added in PR #3929.  The only reference to ActiveFedora::SolrService is in Hyrax::SolrService effectively encapsulating the call to ActiveFedora::SolrService.
